yacl/io/kv/leveldb_kvstore.cc

```diff
@@ -43,7 +43,8 @@ LeveldbKVStore::~LeveldbKVStore() {
         butil::DeleteFile(file_path, true);
       } catch (const std::exception &e) {
         // Nothing we can do here.
-        SPDLOG_INFO("Delete tmp file:{} exception {}", path_, e.what());
+	    std::string e_tmp = e.what();
```

Comment:
OK

Reply:
To make an update, just push a new commit onto your fork＇s branch